dusty/config.py

```diff
@@ -122,9 +123,6 @@ def check_and_load_ssh_auth():
     SSH_AUTH_SOCK environment variable to the current environment.  This allows git clones
     to behave the same for the daemon as they do for the user
     """
-    if os.getenv('SSH_AUTH_SOCK'):
-        logging.info('SSH_AUTH_SOCK already set')
-        return _set_ssh_auth_sock(os.getenv('SSH_AUTH_SOCK'))
```

Comment:
This return statement was doing basically nothing, it could just as easily have been a bare `return`. The updated value of SSH_AUTH_SOCK does not live in the environment, you have to get it through launchctl through the functions below this return. By removing this branch, we run those functions on each command to get the true value of the variable. We then pass the *actual* updated value to `_set_ssh_auth_sock` rather than what's already in our env, which can be stale.